broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -185,6 +190,20 @@ protected void onActorCloseRequested() {
     containers.forEach(ExporterContainer::close);
   }
 
+  @Override
+  protected void handleFailure(final Exception failure) {
+    LOG.error("Actor '{}' failed in phase {} with: {} .", name, actor.getLifecyclePhase(), failure);
+    actor.fail();
+
+    if (failure instanceof UnrecoverableException) {
+      healthStatus = HealthStatus.DEAD;
```

Comment:
To simiplify, you can directly call the listener here and in the else case, and remove the `updateHealthStatus` method.